data/json/professions.json

```
@@ -5762,5 +5768,31 @@
       "male": [ "briefs" ],
       "female": [ "boy_shorts", "sports_bra" ]
     }
+  },
+  {
+    "type": "profession",
+    "id": "officeworker",
+    "name": "Office Worker",
+    "description": "You felt years of your life being sapped away, slaving at your computer as you subsisted on a daily pot of coffee, microwave burritos, and an endless supply of crumpled paper to toss. Perhaps the cataclysm is a refreshing change of pace...",
```

```suggestion
    "description": "You felt years of your life being sapped away, slaving at your computer as you subsisted on a daily pot of coffee, microwave burritos, and an endless supply of crumpled paper to toss.  Perhaps the Cataclysm is a refreshing change of pace…",
```

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md